org.pitivi.Pitivi.json

```diff
@@ -0,0 +1,294 @@
+                    "type": "archive",
+                    "url": "http://downloads.xiph.org/releases/opus/opus-1.1.2.tar.gz",
+                    "sha256": "0e290078e31211baa7b5886bcc8ab6bc048b9fc83882532da4a1a45e58e907fd"
+                }
+            ]
+        },
+        {
+            "name": "frei0r-plugin",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://files.dyne.org/frei0r/releases/frei0r-plugins-1.4.tar.gz",
+                    "sha256": "8470fcabde9f341b729be3be16385ffc8383d6f3328213907a43851b6e83be57"
+                }
+            ]
+        },
+        {
+            "name": "libsoundtouch",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://www.surina.net/soundtouch/soundtouch-1.8.0.tar.gz",
+                    "sha256": "3d4161d74ca25c5a98c69dbb8ea10fd2be409ba1a3a0bf81db407c4c261f166b"
+                }
+            ]
+        },
+        {
+            "name": "lame",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://downloads.sourceforge.net/lame/lame-3.99.5.tar.gz",
+                    "sha256": "24346b4158e4af3bd9f2e194bb23eb473c75fb7377011523353196b19b9a23ff"
+                }
+            ]
+        },
+        {
+            "name": "vo-aacenc",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://downloads.sourceforge.net/project/opencore-amr/vo-aacenc/vo-aacenc-0.1.3.tar.gz",
+                    "sha256": "e51a7477a359f18df7c4f82d195dab4e14e7414cbd48cf79cc195fc446850f36"
+                }
+            ]
+        },
+        {
+            "name": "x265",
+            "cmake": true,
+            "subdir":"source",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://bitbucket.org/multicoreware/x265/downloads/x265_1.9.tar.gz",
+                    "sha256": "3e4654133ed957a98708fdb4cb9a154d9e80922b84e26e43fc462a101c5b15c8"
+                }
+            ]
+        },
+        {
+            "name": "gstreamer",
+            "sources": [
+                {
+                    "type": "git",
+                    "branch": "1.10.5",
+                    "commit": "88ab82933382442a3a626da400f0fd532229b0f9",
+                    "url": "git://anongit.freedesktop.org/gstreamer/gstreamer"
+                }
+            ]
+        },
+        {
+            "name": "gst-plugins-base",
+            "sources": [
+                {
+                    "type": "git",
+                    "branch": "1.10.5",
+                    "commit": "d6f30c28d8cfdad1199f7d29ec248a12bc26606d",
+                    "url": "git://anongit.freedesktop.org/gstreamer/gst-plugins-base"
+                }
+            ]
+        },
+        {
+            "name": "gst-plugins-good",
```

Comment:
In the 3.24 Sdk.

If you only need to build a few extra plugins specify so in the config-opts.

Reply:
Bump as it was rebased to the 3.24 Sdk.

Reply:
We need our own build of Gst as we have backported patches that are not in the 1.12 branch.